natlas-server/app/errors/handlers.py

```diff
@@ -55,3 +55,9 @@ def elastic_unavailable(e):
     err = HTTPError(503, errmsg)
     sentry_sdk.capture_exception(e)
     return build_response(err)
+
+
+@bp.app_errorhandler(NatlasSearchError)
+def invalid_elastic_query(e):
+    sentry_sdk.capture_exception(e)
```

Comment:
One thing we need to think about is how do we differentiate likely client errors from server side errors. I'm okay reporting this just to see, but it would be nice to report this as a warning or include a tag that says it's been classified as a 4xx, not a 5xx.

Reply:
Yeah I mostly wanted to explicitly still capture the exception because I think it'll provide a lot of insight into common search failures, etc. Looks like sentry has a `capture message` concept that might work. In this particular case we can assure that it's a 4xx because NatlasSearchError is currently hardcoded to 400. It's only thrown in a `except elasticsearch.RequestError`, any other failure will still bubble up as a server error.

Reply:
capture_message will lose tons of context about what went wrong. If we wanted to do something else, then we'd need to do capture_event and construct the event ourselves.